src/components/Editable.js

```diff
@@ -69,7 +71,7 @@ export const Editable = connect()(({ isEditing, thoughtsRanked, contextChain, sh
   const thoughtsResolved = contextChain.length ? chain(contextChain, thoughtsRanked) : thoughtsRanked
   const value = head(showContexts ? contextOf(thoughts) : thoughts) || ''
   const thoughtMeta = meta(thoughts)
-  const readonly = thoughtMeta.readonly
+  const readonly = (isRoot(thoughts) || isEM(thoughts)) ? true : thoughtMeta.readonly
```

Comment:
You can simplify to `isRoot(thoughts) || isEM(thoughts) || thoughtMeta.readonly`